tests/modules/apigee_mock_client/src/MockHandlerStack.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * version 2 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301, USA.
+ */
+
+namespace Drupal\apigee_mock_client;
+
+use Apigee\Edge\Exception\ClientErrorException;
+use Drupal\Core\Queue\QueueFactory;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\Psr7\Response;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Symfony\Component\Yaml\Yaml;
+
+class MockHandlerStack extends MockHandler {
+
+  /**
+   * Responses that have been loaded from the response file.
+   *
+   * @var array
+   */
+  protected $responses;
```

Comment:
Because it is an array according to its annotation it should be initialized. Has the type of the array elements already known? If yes the annotation should provide info about that, ex.: `string[]`

Reply:
Done.